### PR TITLE
Apply basic linting to Leo extension JS files

### PR DESCRIPTION
**Pull request by calbach**
Used `eslint --fix` with default Google JS Style rules (with 4sp indent to minimize the diff). My editor was complaining about most lines when I opened this file, and semicolons are generally a good idea in JS.

Mostly this changed the indentation, added semicolons, changed quotes to `'`.

I, the developer opening this PR, do solemnly pinky swear that:

- [x] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/main/resources/jupyter/safe-mode.js [hunk 1]
```diff
@@ -16,87 +16,87 @@ define(() => {
...
-    }
+        'Access-Control-Allow-Headers': '*',
+    };
 
     const basePayload = {
-        mode: "no-cors",
-        headers: headers
-    }
+        mode: 'no-cors',
+        headers: headers,
```
Comment (jdcanas) [inline src/main/resources/jupyter/safe-mode.js]: Trailing commas are not allowed in the JSON spec http://www.json.org/ (from the discussion here: https://stackoverflow.com/questions/201782/can-you-use-a-trailing-comma-in-a-json-object).

Apparently, some browsers will play nice but others won't, so I think it's best to remove it.

**Comment by jdcanas** [comment]
jenkins retest

**Review by jdcanas — CHANGES_REQUESTED** [review]
Looks good overall, though I'd prefer to get rid of the trailing commas.

Not sure why the tests failed, I triggered a retest.

File: src/main/resources/jupyter/safe-mode.js (hunk 1 above)
Comment (calbach) [reply src/main/resources/jupyter/safe-mode.js]: I agree with you stylistically - updated.

To clarify though - JSON != Javascript. JSON is stricter than object definition in Javascript, though the naming might lead one to think they are identical. Trailing commas are valid in Javascript, though incidentally they might cause problems in very old browsers like IE8 (already not supported by AoU or Terra). The SO thread you link to is talking about JSON, not Javascript.

**Comment by jdcanas** [comment]
jenkins retest

**Comment by jdcanas** [comment]
closed for this https://github.com/DataBiosphere/leonardo/pull/1035, didn't want to deal with rebase

**Pull request closed without merging**